Cargo.toml

```diff
@@ -1,39 +1,62 @@
 [package]
-name = "btc_rpc_proxy"
-version = "0.1.1"
-authors = ["Martin Habovštiak <martin.habovstiak@gmail.com>"]
-build = "build.rs"
+authors = [
+  "Martin Habovštiak <martin.habovstiak@gmail.com>",
+  "Aiden McClelland <aiden@start9labs.com>",
+]
 description = "Finer-grained permission management for bitcoind."
 edition = "2018"
+name = "btc-rpc-proxy"
+version = "0.2.2"
+
+[lib]
+name = "btc_rpc_proxy"
+path = "src/lib.rs"
+
+[[bin]]
+name = "btc-rpc-proxy"
+path = "src/main.rs"
 
 [package.metadata.configure_me]
 spec = "config_spec.toml"
 
+[features]
+compat = []
```

Comment:
I suggest renaming to `old_rust` or adding a comment.
We could also check the version in build script and set the feature automatically.